qiskit/chemistry/qubit_transformations/qubit_operator_transformation.py

```diff
@@ -0,0 +1,39 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Base class for transformation to qubit operators for chemistry problems"""
+
+from abc import ABC, abstractmethod
+from typing import Tuple, List
+
+from qiskit.aqua.operators.legacy import WeightedPauliOperator
+from qiskit.chemistry.drivers import BaseDriver
+from qiskit.chemistry.results import StateResult
+
+
+class QubitOperatorTransformation(ABC):
+    """Base class for transformation to qubit operators for chemistry problems"""
+
+    @abstractmethod
+    def transform(self, driver: BaseDriver
+                  ) -> Tuple[WeightedPauliOperator, List[WeightedPauliOperator]]:
+        """transforms to qubit operators """
+        raise NotImplementedError
+
+    @abstractmethod
+    def add_context(self, result: StateResult) -> None:
+        """Adds contextual information to the state result object.
+
```

Comment:
Could it better to keep `interpret` as it was for a sake of similarity? We have `interpret` in the optimization stack and may be in other places.

Reply:
Resolved and about to push.